api/manifest/manifest_test.go

```diff
@@ -177,6 +177,37 @@ worker:
 	_assert(t, cases)
 }
 
+func TestGenerateDockerfileProcfile(t *testing.T) {
+	destDir := mkBuildDir(t, "../../examples/dockerfile-procfile")
+	defer os.RemoveAll(destDir)
+
+	Init(destDir)
+	m, _ := Read(destDir, defaultManifestFile)
+
+	Execer = func(bin string, args ...string) *exec.Cmd {
+		return exec.Command("true")
+	}
+
+	cases := Cases{
```

Comment:
As far as I can tell this is not actually doing what it advertises. It's using the generic Dockerfile from the ruby detection and not generating one from the Procfile.